.eslintrc.js

```diff
@@ -14,32 +18,73 @@ module.exports = {
   },
 
   overrides: [
-    // Enable Prettier
-    {
+    // Anything transpiled with `@kbn/babel-preset/node`
+    serverConfig({
       files: [
-        '.eslintrc.js',
-        'packages/kbn-pm/**/*',
+        'src/**/*.js',
+        'test/**/*.js',
+        'tasks/**/*.js',
+        'utilities/**/*.js',
+        'packages/kbn-pm/**/*.js',
+      ],
+      excludedFiles: ['src/ui/public/**/*.js'],
```

Comment:
I think this needs to exclude anything in a public directory, include public directories in `core_plugins`, so maybe `**/public/**/*.js` is better? Also, maybe we should exclude `**/common/**/*.js` as well? It's transpiled with both but I think the ui restrictions are probably the lowest common denominator.